test/test_aqua.jl

```diff
@@ -13,6 +14,13 @@ include("test_trixi.jl")
                   # in src/solvers/dgmulti/sbp.jl
                   piracies = (treat_as_own = [Trixi.StartUpDG.RefElemData,
                                   Trixi.StartUpDG.MeshData],))
+    @test isnothing(check_no_implicit_imports(Trixi,
+                                              skip = (Core, Base, Trixi.P4est, Trixi.T8code,
+                                                      Trixi.EllipsisNotation)))
```

Comment:
`P4est` and `T8code` are skipped because we use a lot a functions from there, which would make the explicit imports very long. `EllipsisNotation` is skipped because we `@reexport` it.